epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/patchMad.sh

```diff
@@ -50,7 +50,7 @@ touch ${dir}/Events/.keep # this file should already be present (mg5amcnlo copie
 if [ "${patchlevel}" == "2" ]; then
   cd ${dir}
   echo "DEBUG: cd ${PWD}; patch -p4 -i ${scrdir}/MG5aMC_patches/${dir_patches}/patch.common"
-  if ! patch -p4 -i ${scrdir}/MG5aMC_patches/${dir_patches}/patch.common; then status=1; fi  
+  if ! patch -f -p4 -i ${scrdir}/MG5aMC_patches/${dir_patches}/patch.common; then status=1; fi  
```

Comment:
-f, --force
             Forces patch to assume that the user knows exactly what he or she is doing, and to not ask any questions.

This avoids hanging

Reply:
Thanks, but I prefer tha we get an error in case this does not work as expected. I hate silent failures...
https://github.com/madgraph5/madgraph4gpu/pull/882/commits/1bd801d0b59dd03547c7d2234f746f9f60a7cd50

Rephrase: do not assume that I "know exactly what he or she is doing" ;-)